src/main/java/thePackmaster/powers/marisapack/ChargeUpPower.java

```diff
@@ -18,8 +18,11 @@ public class ChargeUpPower extends AbstractPackmasterPower implements CloneableP
     public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
     public static int THRESHOLD = 6, MODIFY_AMT = 2;
 
+    public static int chargeUpGainThisCombat = 0;
+
     public ChargeUpPower(int amount) {
         super(POWER_ID, NAME, PowerType.BUFF, false, Wiz.p(), amount);
+        chargeUpGainThisCombat += amount;
```

Comment:
I recall having logic like this in the constructor causing issues with Together in Spire. Not the biggest of compatibility issues, but it might be better to have this logic in both `onInitialApplication` and `stackPower` instead.

Reply:
That was my original approach, but I figured I'd keep it the same as Mantra. Guess I'll switch back